config/locales/en/settings.yml

```diff
@@ -207,3 +207,14 @@ en:
         max_size_description: "Maximum size allowed for an uploaded document (in Megabytes/MB)"
         content_types: "Accepted content types for documents"
         content_types_description: "Select all the content types allowed for uploaded documents"
+    sdg:
+      process_debates: Linking debates to SDG
```

Comment:
Maybe we should ask whether there's an official UN terminology here :thinking:.

Reply:
I saw that they used the word "Related". But I liked it better "Linking debates to SDG" than "Relate debates to SDG"